run/static_outbound/main.tf

```diff
@@ -109,40 +109,34 @@ resource "google_compute_router_nat" "default" {
 # [END cloudrun_service_static_nat]
 
 # [START cloudrun_service_static_service]
-resource "google_cloud_run_service" "default" {
+resource "google_cloud_run_v2_service" "default" {
   provider = google-beta
   name     = "cr-static-ip-service"
   location = google_compute_subnetwork.default.region
 
   template {
-    spec {
-      containers {
-        # Replace with the URL of your container
-        #   gcr.io/<YOUR_GCP_PROJECT_ID>/<YOUR_CONTAINER_NAME>
-        image = "us-docker.pkg.dev/cloudrun/container/hello"
-      }
+    containers {
+      # Replace with the URL of your container
+      #   gcr.io/<YOUR_GCP_PROJECT_ID>/<YOUR_CONTAINER_NAME>
```

Comment:
should we show an example of an AR registry url?

Reply:
More holistic AR updates planned separately